engines/python/setup/djl_python/deepspeed.py

```diff
@@ -95,11 +95,11 @@
 
 
 def get_torch_dtype_from_str(dtype: str):
```

Comment:
Do we want to include something like this https://github.com/microsoft/DeepSpeed-MII/blob/main/mii/config.py#L7?

Or do we need to add documentation about what the supported dtype strings are in serving.properties?